tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -275,3 +279,66 @@ TEST(RDFHelpers, SaveGraphToFile)
 
    gSystem->Unlink(outFileName);
 }
+
+TEST(RunGraphs, RunGraphs)
+{
+#ifdef R__USE_IMT
+   ROOT::EnableImplicitMT();
+#endif // R__USE_IMT
+
+   ROOT::RDataFrame df1(3);
+   auto df1a = df1.Define("x", "float(rdfentry_)");
+   auto r1 = df1a.Sum("x");
+   auto r2 = df1a.Count();
+
+   ROOT::RDataFrame df2(3);
+   auto df2a = df2.Define("x", "2.f * float(rdfentry_)");
+   auto r3 = df2a.Sum("x");
+   auto r4 = df2a.Count();
+
+   std::vector<RResultHandle> v = {r1, r2, r3, r4};
+   ROOT::RDF::RunGraphs(v);
+
+   EXPECT_EQ(df1.GetNRuns(), 1u);
+   EXPECT_EQ(df2.GetNRuns(), 1u);
+
+   for (auto& h : v) EXPECT_TRUE(h.IsReady());
```

Comment:
needs clang-formatting. I understand you prefer putting the body of for loops and ifs on the same line if it's short but it's not our coding style :sweat_smile: (and it's easier to miss for the reader, especially one like me that's not used to this style)

Reply:
:+1: Done!